pad_server/src/state.c

```diff
@@ -52,3 +53,15 @@ int padstate_actuate(padstate_t *state, uint8_t id, bool new_state) {
 
     return pthread_rwlock_unlock(&state->rw_lock);
 }
+
+int padstate_get_actuator(padstate_t *state, uint8_t act_id, bool *act_val) {
```

Comment:
Nit but I'd call this `padstate_get_actstate` or something since you're not getting the actual actuator, just it's
state.